setup.py

```diff
@@ -41,6 +41,9 @@
         "pandas>=1.5.3",
         "fasteners",
         "GitPython>=3.1.40",
+        # Necessary until upstream packages account for the breaking
+        # change to numpy
+        "numpy<2.0.0",
```

Comment:
Not specific to this update; we may need this change on `main` if we want to update the tkml code. Ultimately, its ORT who needs to make this change though.

Reply:
Nice! We've been doing this on all of our projects while our dependencies catch up with this themselves.